src/components/directory-content.tsx

```diff
@@ -10,7 +10,6 @@ import { useDispatch, useSelector } from 'react-redux';
 import { FormattedMessage, useIntl } from 'react-intl';
 import { Box, Button, CircularProgress, Grid, SxProps, Theme } from '@mui/material';
 import {
-    CriteriaBasedFilterEditionDialog,
```

Comment:
CriteriaBasedFilterEditionDialog is not used anymore 
to be removed

Reply:
I don't understand ! where i should remove that from ?

Reply:
in commons-ui

Reply:
I think it is already the case